_posts/0000-01-02-ibregg.md

```diff
@@ -0,0 +1,6 @@
+---
+ layout : titolo della diapositiva : "Benvenuti nella nostra seconda diapositiva"
+ ---
+Il tuo testo
+Usa la frecia sinistra per tornare indietro!
```

Comment:
È molto facile scrivere alcune parole in ** grassetto ** e altre in * corsivo * con Markdown. Puoi anche [link to Google!] (Http://google.com)